docs/snippets/snippets-policies.md

```diff
@@ -168,6 +169,13 @@ The `defaults` section of the policy file is...
 | `accessKeyAssetId` | `string`<uuid> or `null` non-empty | ID of credentials asset of type access-key, for private S3 buckets. |
 | `url` | `string` or `null` non-empty | The url of the S3 service provider. The default is the URL of the Amazon AWS S3 service. |
 
+#### ConfigMap Resource Description Fields
+
+|Field | Type | Description |
+| -- | -- | --|
+|`json:"configMap"` | `string` | The name of the ConfigMap resource. ConfigMap is mandatory for creating a workspace. |
+|`json:"mountPath"` | `string` | Local path within the workspace to which the PVC bucket will be mapped. ClaimName is mandatory for creating a workspace. |
```

Comment:
`Local path within the workspace to which the ConfigMap will be mapped`

Reply:
```suggestion
|`json:"mountPath"` | `string` | Local path within the workspace to which the ConfigMap bucket will be mapped. ClaimName is mandatory for creating a workspace. |
```